code/game/objects/effects/decals/remains.dm

```diff
@@ -27,12 +27,26 @@
 	desc = "They look like the remains of a small reptile."
 	icon_state = "lizard"
 
-/obj/effect/decal/remains/attack_hand(mob/user as mob)
-	to_chat(user, "<span class='notice'>[src] sinks together into a pile of ash.</span>")
-	var/turf/simulated/floor/F = get_turf(src)
+//Target turns to ash.
+/proc/crumble(user, source)
+	to_chat(user, "<span class='notice'>[source] sinks together into a pile of ash.</span>")
+	var/turf/simulated/floor/F = get_turf(source)
```

Comment:
```suggestion
/obj/effect/decal/remains/proc/crumble()
	visible_message(SPAN_NOTICE("[source] sinks together into a pile of ash."))
	var/turf/simulated/floor/F = get_turf(src)
	if (istype(F))
		new /obj/effect/decal/cleanable/ash(F)
```
We only want this proc to pertain to `/obj/effect/decal/remains`. Doing just /proc makes it global. Also this way no need to designate "source"
<span> edit is just cleanup.
visible_message will show to everyone. To_chat will only show to the person you had defined as pulledby. So we can also delete `user` from your proc
and by moving the /proc to `/obj/effect/decal/remains` we can use `src`